VATSpy.dat

```diff
@@ -18059,7 +17942,7 @@ RKRR-S|Incheon ACC (South) - Incheon||RKRR-S
 RKDA|Daegu ACC - Incheon||RKDA
 RKDA-E|Daegu ACC (East) - Incheon||RKDA
 RKDA-C|Daegu ACC (Central) - Incheon||RKDA
-RJBG|Kobe ACC - Fukuoka||RJBG
+RORG|Naha ACC - Fukuoka||RORG

```

Comment:
need to look at this as well I am not shore if the user has an old version on there end